clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -626,79 +687,94 @@ static Expected<StringRef> runSYCLPostLink(ArrayRef<StringRef> InputFiles,
     CmdArgs.push_back(File);
   if (Error Err = executeCommands(*SYCLPostLinkPath, CmdArgs))
     return std::move(Err);
-  return *TempFileOrErr;
-}
 
-// This table is used to manage the output table populated by sycl-post-link.
-struct Table {
-  struct SYCLTableEntry {
-    std::string IRFile;
-    std::string PropFile;
-    std::string SymFile;
-  };
+  if (DryRun) {
```

Comment:
Can you please add a comment here? Thanks

Reply:
Added.